### PR TITLE
Swagger Auth Header Code Generation Updates

### PR DESCRIPTION
**Pull request by atulsriv**

### CODE REVIEW

File: proto/api.proto [hunk 1]
```diff
@@ -9,6 +9,16 @@ import "google/api/visibility.proto";
...
+  security_definitions: {
+    security: {
+      key: "Authorization";
+      value: {
+        type: TYPE_API_KEY;
+        in: IN_HEADER;
+        name: "Authorization";
+      };
+    }
+  };
```
Comment (aspin) [inline proto/api.proto:29]: I think there's another `security` field you can include (on same level as `security_definitions`) to apply it global: https://swagger.io/docs/specification/authentication/

something like
```
  security: [
    {
      security_requirement: {
        key: "Authorization";
        value: {
          scope: []
        }
      }
    }
  ]
```
?

File: proto/api.proto [hunk 2]
```diff
@@ -9,6 +9,14 @@ import "google/api/visibility.proto";
 import "protoc-gen-openapiv2/options/annotations.proto";
 
 option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+  security: {
+    security_requirement: {
+      key: "Authorization";
```
Comment (aspin) [inline proto/api.proto]: nit: probably "Auth Header" would be more verbally consistently with other bloxroute docs. would also help distinguish which field is the actual header name and which field is the label that shows up in Swagger

**Review by aspin — APPROVED** [review]

File: proto/api.proto [hunk 3]
```diff
@@ -36,6 +55,11 @@ service Api {
       get: "/api/v1/market/price"
     };
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      security: {
```
Comment (aspin) [inline proto/api.proto]: think these lines below are unnecessary

Comment (atulsriv) [reply proto/api.proto]: yeah, been updated

**Review by aspin — APPROVED** [review]

**Pull request merged**